pkg/hooks/webhook.go

```diff
@@ -159,6 +165,18 @@ func (w *webhookExecutor) Call(webhookRequest api.WebhookRequest, webhookRespons
 	}
 	defer response.Body.Close()
 
+	if response.StatusCode == 429 {
+		var afterSecond int
+		retryAfter := response.Header.Get("Retry-After")
+		nextTime, err := time.Parse(time.RFC1123, retryAfter)
+		if err != nil {
+			afterSecond, _ = strconv.Atoi(retryAfter)
```

Comment:
@weijiany how it will behave if there is unparsable value ?

Reply:
The value of `afterSecond` will be 0. I could set a default value for this, but I don't want to handle the error that is parsed. Because the HTTP protocol makes us if the response code is 429, the `Retry-After` only can support two kinds.

Reply:
got it, so basically it will behave as when no option set, got it.